### PR TITLE
[DEVELOP-1316] Assorted UX improvements based on user feedback

### PR DESCRIPTION
**Pull request by trydionel**
**Changes:**
* Filter by project
* Exclude done records
* Better issue sorting
* Improve card layout

**Screenshot of revised importer:**

Screen Shot 2022-03-07 at 10 36 35 AM

### CODE REVIEW

File: src/importers/azureDevops.js [hunk 1]
```diff
@@ -45,6 +45,12 @@ async function sendGetHttpRequest(endpoint, forWhat) {
         value: organization.accountName,
       }));
       return organizationInfo;
+    case "projectInfo":
```
Comment (trydionel) [inline src/importers/azureDevops.js]: This pattern isn't ideal. Should we refactor this now?

Comment (percyhanna) [reply src/importers/azureDevops.js]: It does seem a little odd. Normally there would just be a simpler `get` method to execute the request itself, and then light wrapper methods like `getUser`, etc.

I see methods like that exist below, I would expect these transformations to occur in those methods, not here.

File: src/importers/azureDevops.js [hunk 2]
```diff
@@ -63,9 +69,27 @@ export async function getOrganizationInfo() {
...
+  return result;
+}
+
+export async function getWorkItems(organization, project, offset) {
   const body = {
-    query: "Select  [System.Id], [System.Title], [System.State] From WorkItems",
+    query: `
+      Select  [System.Id], [System.Title], [System.State]
+      From WorkItems
+      Where [State] <> 'Done'
```
Comment (percyhanna) [inline src/importers/azureDevops.js]: Is `Done` an internal value/key, or is it configurable by the user?

File: src/importers/azureDevops.js [hunk 3]
```diff
@@ -63,9 +69,27 @@ export async function getOrganizationInfo() {
...
+      And [System.TeamProject] = '${project}'
+      Order by [Microsoft.VSTS.Common.Priority] asc, [System.CreatedDate] desc
```
Comment (percyhanna) [inline src/importers/azureDevops.js:77]: The square brackets are giving me flashbacks to SQL Server/MS Access from 21 years ago…

Comment (trydionel) [reply src/importers/azureDevops.js:77]: Microsoft Access! I haven't thought about that in a long time. 😆

File: src/importers/azureDevops.js (hunk 2 above)
Comment (trydionel) [reply src/importers/azureDevops.js]: It is user-configurable, yea... but I don't see a way to filter by status category with ADO's query language 😕

File: src/importers/azureDevops.js (hunk 1 above)
Comment (trydionel) [reply src/importers/azureDevops.js]: Agreed. I'll revise this.

File: src/importers/azureDevops.js (hunk 2 above)
Comment (trydionel) [reply src/importers/azureDevops.js]: Found it! Status categories aren't a separate field. They're available via a separate operator on the status field.

**Review by percyhanna — APPROVED** [review]

**Pull request merged**